provider/resource_keycloak_realm_test.go

```diff
@@ -429,6 +488,76 @@ resource "keycloak_realm" "realm" {
 	`, realm, realmDisplayName)
 }
 
+func testKeycloakRealm_WithSmtpServer(realm, host string, from string) string {
+	return fmt.Sprintf(`
+resource "keycloak_realm" "realm" {
+	realm        = "%s"
+	enabled      = true
+	display_name = "%s"
+	smtp_server {
+      host = "%s"
+	  port = 25
+	  from_display_name = "Tom"
+      from = "%s"
```

Comment:
would you be able to fix the indentation for these SMTP config blocks?

Reply:
strange, I do not have this weird indentation, I will copy and paste the whole block again